### PR TITLE
Blog building day1 5 19 16

### PR DESCRIPTION
**Pull request by kurtcunningham**
Blog post on building for day one and what it means.

Ready to merge and then delete branch.

### CODE REVIEW

File: source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: Build Your Product For Day One
+date: 2016-05-19 22:02 UTC
+tags: user experience, design
+authors: Kurt Cunningham
+---
+
+At Made By Munsters we work with companies of all sizes. Some are brand new start-ups. Others are well established, long-standing businesses. Yet, we treat all compaines the same when hired to design and or develop their products.
```
Comment (nearbycoder) [inline source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown]: misspelling */compaines/*

File: source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+Our process _always_ starts with an explaination of our design and development goal.
```
Comment (nearbycoder) [inline source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown]: misspelling */explaination/*

File: source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+That goal: Build for day one. Don't focus on what's to come a year from now or even three months from now. Focus on your first release.
+
+Our team looks at the immediate needs and problems our client may be facing and then as a team we define a solution that best fits the first releases's needs.
```
Comment (nearbycoder) [inline source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown]: releases is already plural not sure if the *'s* is needed.

File: source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+<br />
+
+##### Defining Day One
+
+What do we mean by day one?
+
+At Made By Munsters, we are not big fans of MVPs or minimal viable products. To us, those types of products don't solve the problems that cleints hired us to solve. They scratch the surface and allow a company to test an idea, but they don't solve the problem.
```
Comment (nearbycoder) [inline source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown]: */cleints/* misspelled

File: source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+Instead we build proven solutions for the first release.
+
+These solutions address the issues, we as a team, have identified as our problems early on during product's inception phase.
```
Comment (nearbycoder) [inline source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown]: rephrase: "These solutions address the issues, we as a team, have identified as problems early on during product's inception phase". <- removed the *our*

File: source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+How do we know a solution will work or consider a solution a proven method? We do this through early phase testing. By getting a product in front of users we can test theories, tweak features and release a solution that meets our client's needs right out the gate.
+
+This is why we refer to our products as day one releases rather than MVPs.
+
+<br />
+
+##### Developing A Day One Release
+
+In order to develop a full-scale solution for day one, there are a serveral key steps that must be completed. At Made By Munsters, we found that following this process has led to best outcomes.
```
Comment (nearbycoder) [inline source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown]: misspelling */serveral/*

File: source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+>"To truly solve a problem, one must fully understand the problem they are solving."
+
+Well, maybe someone didn't say those exact words. But I am sure someone made similar remarks at some point in history.
+
+The goal of this step: Understand what the product will solve. Research platforms that solve similar problems. Then, establish and determine how this particular product will differentiate itself from others in the marketplace.
+
+Once the client and your team are on the same page, write down this goal. Don't lose track of it. Pin it up to a wall. Make it your desktop background. Do whatever you need to do to stay focused on it.
+
+This is the most cruical step in the entire process. As products grow and scopes change, this thesis should remain constant. If it does not, you will need to re-elvauate what you're building.
```
Comment (nearbycoder) [inline source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown]: misspellings: */cruical, re-elvauate/*

File: source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+To build a truely successful day one product, it's important to stay true to your origianl assumption. At least in some shape or form.
```
Comment (nearbycoder) [inline source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown]: misspellings: */truely, origianl/*

File: source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+
+##### Solve Problems, Don't Create Them
+
+We have said this before on our blog - [Crafting Our UX Process](/blog/posts/crafting-our-ux-process.html) - but it's important to solve problems that exist. Don't create problems or find solutions for issues that have not been voiced.
+
+This is easier said then done. But if you have done your homework, interviewed your users and have a clear thesis statement, you should be able to avoid these unnecessary features.
+
+When you solve for problems you anticipate, rather than find solutions for what's right in front of you, you will find you fail to test and complete core functionality.
+
+We've made this miskate many times. I am sure all design and development studios have at some point in their history.
```
Comment (nearbycoder) [inline source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown]: misspelling: *miskate*

File: source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+
+As clients ask for more and more features without adjusting timelines, you rush and haphazardly deploy features. This must be avoided.
+
+It's best to discuss and analyze what features make sense and what can wait. We ask for data and or case studies to help us understand why these new features might be needed for day one.
+
+If that type of information is unavailable, then wait for it. More often then not, you will create a solution for a problem users did not need soloved.
```
Comment (nearbycoder) [inline source/blog/posts/2016-05-19-build-with-day-1-in-mind.html.markdown]: misspelling: *soloved*

**Comment by nearbycoder** [comment]
Altogether great blog post and helpful to understand the difference between Day one and MVP. Pointing out that we want to see data and or case studies before a feature is implemented for day one shows that we put the clients needs first and want to help guide them into the right day one solution! Just a few spelling issues and changing the blog date, then it's good to go!

**Comment by kurtcunningham** [comment]
Fixed the spelling errors and updated the date. Have a look and merge if we are good to go.

**Pull request merged**